ionic.config

```diff
@@ -0,0 +1,7 @@
+{
+  "lastVersionChecked": "1.7.14",
```

Comment:
Yes, It's necessary because if you don't have this file the IONIC server will ask which address it should run (localhost or something else).